app/models/school_submission.rb

```diff
@@ -60,8 +60,6 @@
 
 class SchoolSubmission < ActiveRecord::Base
 
-  before_validation :validate_page, on: [:create, :update]
```

Comment:
Since we aren't depending on lifecycle methods (instead we call our own method), we don't need these anymore!